Plugins/GeoModel/include/Acts/Plugins/GeoModel/detail/GeoModelDbHelper.hpp

```diff
@@ -0,0 +1,40 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2024 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+namespace Acts::detail::GeoModelDbHelper {
+
+/// @brief Split a string into a vector of strings
+///
+/// @param entry the string to be split
+/// @param deliminater split indicator
+///
+/// @return a vector of strings
+inline std::vector<std::string> tokenize(const std::string& entry,
```

Comment:
boost provides `boost::split` (https://www.boost.org/doc/libs/1_85_0/doc/html/string_algo/usage.html#id-1.3.3.5.9) which does exactely what this does, maybe we should use this instead of writing an own function?

Reply:
I know, but eventually we would like to reduce BOOST dependency ... not sure though.

Reply:
Ah I didn't know we try to do that... I would say we have mostly pushed out dependence on compiled boost libraries (`filesystem`, `program_options`), but I'm not sure we can & should try to get rid of boost header-only functionality?
IMO it is the whole point of boost to provide stable and fast implementations of standard functionality...